From khali@linux-fr.org Wed Oct 26 12:14:13 2005
Date: Wed, 26 Oct 2005 21:14:16 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Henk Vergonet <henk@god.dyndns.org>, Mark McClelland <mark@alpha.dyndns.org>
Subject: [PATCH 05/16] i2c: ID redefinition cleanups
Message-Id: <20051026211416.293d8e42.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-id-clean-redefinitions-1.patch

Fix several redefinitions of i2c IDs. i2c IDs must not be defined
outside of i2c-id.h.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Henk Vergonet <henk@god.dyndns.org>
Acked-by: Mark McClelland <mark@alpha.dyndns.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/busses/i2c-prosavage.c |    5 -----
 include/media/ovcamchip.h          |   14 --------------
 2 files changed, 19 deletions(-)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-prosavage.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-prosavage.c
@@ -83,11 +83,6 @@ struct s_i2c_chip {
 /*
  * i2c configuration
  */
-#ifndef I2C_HW_B_S3VIA
-#define I2C_HW_B_S3VIA	0x18	/* S3VIA ProSavage adapter		*/
-#endif
-
-/* delays */
 #define CYCLE_DELAY	10
 #define TIMEOUT		(HZ / 2)
 
--- gregkh-2.6.orig/include/media/ovcamchip.h
+++ gregkh-2.6/include/media/ovcamchip.h
@@ -17,20 +17,6 @@
 #include <linux/videodev.h>
 #include <linux/i2c.h>
 
-/* Remove these once they are officially defined */
-#ifndef I2C_DRIVERID_OVCAMCHIP
-	#define I2C_DRIVERID_OVCAMCHIP	0xf00f
-#endif
-#ifndef I2C_HW_SMBUS_OV511
-	#define I2C_HW_SMBUS_OV511	0xfe
-#endif
-#ifndef I2C_HW_SMBUS_OV518
-	#define I2C_HW_SMBUS_OV518	0xff
-#endif
-#ifndef I2C_HW_SMBUS_OVFX2
-	#define I2C_HW_SMBUS_OVFX2	0xfd
-#endif
-
 /* --------------------------------- */
 /*           ENUMERATIONS            */
 /* --------------------------------- */
